packages/site-components/src/SearchInput/index.tsx

```diff
@@ -25,7 +34,9 @@ export function SearchInput() {
   }, [searchTerm, searchResults]);
 
   useEffect(() => {
```

Comment:
This effect should be removed and the code moved into the `handleSearch` callback so we are not using an effect to fetch data